pkg/virt-handler/virtwrap/schema.go

```diff
@@ -55,14 +55,21 @@ func init() {
 }
 
 const (
-	NoState     LifeCycle = "NoState"
-	Running     LifeCycle = "Running"
-	Blocked     LifeCycle = "Blocked"
-	Paused      LifeCycle = "Paused"
-	Shutdown    LifeCycle = "Shutdown"
-	Shutoff     LifeCycle = "Shutoff"
-	Crashed     LifeCycle = "Crashed"
-	PMSuspended LifeCycle = "PMSuspended"
+	NoState      LifeCycle = "NoState"
+	Running      LifeCycle = "Running"
+	Blocked      LifeCycle = "Blocked"
+	Paused       LifeCycle = "Paused"
+	Shutdown     LifeCycle = "Shutdown"
+	Shutoff      LifeCycle = "Shutoff"
+	Crashed      LifeCycle = "Crashed"
+	PMSuspended  LifeCycle = "PMSuspended"
+	Unknown      LifeCycle = "Unknown"
+	User         LifeCycle = "User"
```

Comment:
We have to somehow avoid the ambiguity between "Shutdown" and "User".

I have a feeling that treating "Shutdown" always as a shutdown from withing the VM might be a good thing. Then we can treat Shutoff/Shutdown as "Shutoff" only. All other reasons for "Shutoff" can be directly mapped to the LifeCycle.